lib/utils/helpers.js

```diff
@@ -13,4 +13,5 @@ export const intersperse = (list, separator) => {
 
 export const getRandomInt = limit => Math.floor(Math.random() * limit)
 
-export const getElementFromSeed = (list, seed) => list[seed % list.length]
+export const getElementFromSeed = (list, seed, seedLimit) =>
+  list[Math.round((seed * (list.length - 1)) / (seedLimit - 1))]
```

Comment:
Could you add test for this function? See `lib/routes.test.js` as an example of how to write them. Example of running one test file:
```bash
npm tape lib/routes.test.js
```

I can't follow how this works by reading the code and a few input and output examples would help.